service/header/header_verify.go

```diff
@@ -96,6 +64,28 @@ func (eh *ExtendedHeader) VerifyAdjacent(untrst *ExtendedHeader) error {
 	return nil
 }
 
+// clockDrift defines how much new header's time can drift into
+// the future relative to the now time during verification.
+var clockDrift = 10 * time.Second
+
+// verify performs basic verification of untrusted header.
+func (eh *ExtendedHeader) verify(untrst *ExtendedHeader) error {
+	if untrst.ChainID != eh.ChainID {
+		return fmt.Errorf("new untusted header has different chain %s, not %s", untrst.ChainID, eh.ChainID)
+	}
+
+	if !untrst.Time.After(eh.Time) {
+		return fmt.Errorf("expected new untrusted header time %v to be after old header time %v", untrst.Time, eh.Time)
+	}
+
+	now := time.Now()
+	if !untrst.Time.Before(now.Add(clockDrift)) {
+		return fmt.Errorf("new untusted header has a time from the future %v (now: %v)", untrst.Time, now)
```

Comment:
Can you add the maximum block drift to this error message? Otherwise the user will have to dig into the code to figure out what it is.

Reply:
Ok